Allan_Cannon/OOP/MathDojo.rb

```diff
@@ -0,0 +1,42 @@
+class MathDojo
+  @val
+  attr_reader
+
+  def initialize
+    @val = 0
+  end
+
+  def add *nums
+    for num in nums
```

Comment:
This is great, but it also looks odd in ruby to write out a for loop this way. It would be cool if you could implement this same functionality but with a ruby iterator